WalletWasabi.Tests/UnitTests/WabiSabi/Client/ArenaClientTests.cs

```diff
@@ -83,5 +84,55 @@ public async Task RegisterInputAsyncTest()
 			Assert.Single(apiClient.AmountCredentialClient.Credentials.Valuable, x => x.Amount.ToMoney() == reissuanceAmounts.First());
 			Assert.Single(apiClient.AmountCredentialClient.Credentials.Valuable, x => x.Amount.ToMoney() == reissuanceAmounts.Last());
 		}
+
+		[Fact]
+		public async Task SignTransactionAsync()
```

Comment:
nit: perhaps it's cleaner having two test methods, one for the happy path and one for the various error conditions?

Reply:
It is fine in this case., not too long.